dom.bs

```diff
@@ -1789,6 +1792,7 @@ to <a for=AbortSignal>signal abort</a> on <a>this</a>'s <a for=AbortController>s
 interface AbortSignal : EventTarget {
   [NewObject] static AbortSignal abort(optional any reason);
   [Exposed=(Window,Worker), NewObject] static AbortSignal timeout([EnforceRange] unsigned long long milliseconds);
+  [NewObject] static AbortSignal _any(sequence&lt;AbortSignal> signals);
```

Comment:
I'm somewhat surprised that `Promise.any` takes an iterable and not rest parameters.

Reply:
I think the idea is array in, array out.

Reply:
I don't think that's what `Promise.any` does, but maybe the other statics do and this matches those for consistency.

Reply:
oh yeah, you're right. I read `Promise.all`.